paddle/fluid/framework/tensor.h

```diff
@@ -106,7 +106,7 @@ class Tensor {
    *       this field.
    */
 
-  mkldnn::memory::format_tag format_ = mkldnn::memory::format_tag::undef;
+  dnnl::memory::format_tag format_ = dnnl::memory::format_tag::undef;
```

Comment:
@XiaoguangHu01  @chenwhql  @zhiqiu  @Xreki  @luotao1
These are the only changes made to tensor.h. Please approve this simple refactor.

Reply:
Will the concept of mkldnn be deprecated in the future? Do we need to change the kMKLDNN of the DataLayout to kDNNL? and do the naming of some MKLDNN functions and directories need to be changed?

Reply:
The names of these elements don't NEED to be changed for the framework to properly work with future oneDNN versions, but I feel like changing them would still be healthy and would eliminate some confusion related to the naming. Having said that, changing them is definitely not of high priority right now.

Reply:
I see, thanks